scripts/release.js

```diff
@@ -156,6 +156,19 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install the component app as dependency and build the test app to ensure no errors or build issues
+process.chdir(path.join(__dirname, '..'));
+spawnOrFail('yalc', ['publish']);
+process.chdir(path.join(__dirname, '../..'));
+spawnOrFail('npx', ['create-react-app dependency-check-app']);
+process.chdir(path.join(__dirname, '../../dependency-check-app'));
+spawnOrFail('yalc', ['add amazon-chime-sdk-component-library-react']);
```

Comment:
This does not actually install the package so you need to manually call npm install amazon-chime-sdk-component-library-react

Reply:
Can I just do a "npm install" ? It think it will work in the same way,

Reply:
doing 'npm install amazon-chime-sdk-component-library-react' would try to install it from NPM right? I think once you add it, you can just do npm install.

Reply:
yes, in my pr, I use npm install